mqtt-endpoint/src/service/session/mod.rs

```diff
@@ -27,6 +27,50 @@ use std::{
     sync::Arc,
 };
 
+#[derive(Clone)]
+pub enum Dialect {
+    DrogueV1,
+    LongTopicDialect,
```

Comment:
I would rename this `PlainTopic`. Drop the `Dialect` suffix, as these are all dialects and it just is the plain topic.